### PR TITLE
Format SQL queries so they are easier to read.

### PR DESCRIPTION
**Pull request by markstory**
Use a library to format SQL queries into a more readable formatting.

:notebook: This does not include syntax highlighting.

Refs #349

### CODE REVIEW

**Comment by AD7six** [comment]
Is it worth making that configurable, i.e. so the user can do nothing/whitespace/syntax highlighting as they see fit?

File: src/Template/Element/sql_log_panel.ctp [hunk 1]
```diff
@@ -63,7 +63,7 @@ $noOutput = true;
             <tbody>
                 <?php foreach ($queries as $query): ?>
                 <tr>
-                    <td><?= h($query['query']) ?></td>
+                    <td><pre><?= SqlFormatter::format($query['query'], false) ?></pre></td>
```
Comment (ndm2) [inline src/Template/Element/sql_log_panel.ctp]: This should stil use `h()`, as entity encoding is only being done when using syntax highlighting.

Comment (markstory) [inline src/Template/Element/sql_log_panel.ctp]: OK. Given that @lorenzo thinks adding syntax highlighting is important, I'll be adding that tonight.

**Comment by markstory** [comment]
Queries have colours now.

**Comment by lorenzo** [comment]
Fantastic! :clap:

**Pull request merged**